src/configuration/evars.md

```diff
@@ -246,6 +246,16 @@ A boolen variable to show UTF-8 characters instead of ANSI.
 
 Enables or disables "fortune" messages displayed at each rizin start.
 
+### cfg.fortunes.file
+
+Rizin has two types for fortunes: tips and fun. Fortunes of the type 'tips' are general tips to help you use Rizin better, whereas the other one prints some lighthearted jokes. You can choose which type of fortune to display, using this variable.
+
+Rizin also supports custom fortunes. You can save your fortunes in a file and provide the relative or absolute path to the same variable, to have them displayed at the startup.
+
+```
+[0x00000000]> e cfg.fortunes.file = /path/to/my/fortunes.txt
```

Comment:
Probably this is my mistake, but actually there must not be any space between the `=`.
so it should be: `e cfg.fortunes.file=/path/to/my/fortunes.txt`